pkg/filesharecontroller/controller.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Change all copyright header to "2019 OpenSDS Authors"

Reply:
Done